pkg/engine/exceptions.go

```diff
@@ -53,6 +55,20 @@ func matchesException(
 		resource = policyContext.OldResource()
 	}
 	for _, candidate := range candidates {
+		if rule.HasVerifyImages() || rule.HasValidatePodSecurity() {
+			if candidate.Spec.HasImages() {
+				for _, iv := range rule.VerifyImages {
+					for _, iref := range iv.ImageReferences {
+						for _, i := range *candidate.Spec.Images {
+							if wildcard.Match(iref, i) {
+								return candidate, nil
+							}
+						}
+					}
+				}
+				return nil, errors.New("no image is matched")
```

Comment:
So this is the logic if an image matches in `verifyImage` rules, what about the `validate.podSecurity` rules ?

Reply:
For this I had a question: Is it possible for a user to define both `verifyImage` and `valdiate.podSecurity`?

Reply:
I have pushed a commit that changes the logic. Just wondering if the logic would satisfy the use case if a user has defined both the rules